### PR TITLE
Update/#286 about us page update

### PR DESCRIPTION
**Pull request by daniel-cy-lu**
About Us page responsive css completed.

Patrick's Figma mock up https://www.figma.com/file/fXh58MfxEDh1ssgxQyPb5f/Overture_Refresh?type=design&node-id=1167-5674&mode=design&t=kV2MP4uxoGMHia75-0

### CODE REVIEW

File: src/pages/about-us/index.js [hunk 1]
```diff
@@ -1,104 +1,114 @@
...
+                title="Extensible"
+                img_src="aboutUsExtensible"
+                alt="Extensible Icon"
+                text="The components should be designed to be expandable and customizable to meet the specific needs of researchers."
+              />
+              <YellowButton
+                title="Reusable"
+                img_src="aboutUsReusable"
+                alt="Reusable Icon"
+                text="The components should be designed to be expandable and customizable to meet the specific needs of researchers."
```
Comment (MitchellShiell) [inline src/pages/about-us/index.js:70]:
```suggestion
                text="The components should be designed for use in different contexts and with minimal modifications."
```

**Comment by MitchellShiell** [comment]
@patrickdsantos how should we better differentiate the links at the bottom from the icons in the middle

**Comment by ciaranschutte** [comment]
code is looking good.

@daniel-cy-lu just need to fix merge conflicts + update text from Mitchells message then we're good to go !

**Review by ciaranschutte — APPROVED** [review]

**Comment by patrickdsantos** [comment]
@MitchellShiell @daniel-cy-lu We can underline the titles on the Connect with Our Community section!

**Pull request merged**